docs/api.md

```diff
@@ -3397,6 +3397,7 @@ await page.mouse.up();
 
 <!-- GEN:toc -->
 - [mouse.click(x, y[, options])](#mouseclickx-y-options)
+- [mouse.currentState()](#mousecurrentstate)
```

Comment:
Why not mouse.position?

Reply:
The idea was to leave some space for other state parameters, e.g. buttons if/when they are requested.

I'm fine with either option.